### PR TITLE
Add geocode.earth production and dev endpoints

### PR DESCRIPTION
**Pull request by orangejulius**
This replaces the Mapzen Search prod_build and dev endpoints with production and dev for [geocode.earth](https://geocode.earth).

A new API key with a small rate limit for geocode.earth has been created specifically for the compare app.

For now, production Mapzen Search remains as the first endpoint, since it is still up. However, it will no doubt soon go away.

Connects https://github.com/pelias/pelias/issues/703

### CODE REVIEW

**Comment by missinglink** [comment]
I think you'll also need to bump `var currentVersion = xxx`.

If I remember correctly, I set up a version number so that, your localStorage values will be used so long as the `currentVersion` is the same, if the server serves a document with a higher `currentVersion` than in your localStorage then your localStorage will be replaced with the new default values.

tl;dr we need the `currentVersion++` or existing users existing localStorage won't be overridden by the new defaults.

**Review by missinglink — APPROVED** [review]
provisionally approved, pending version bump

**Comment by orangejulius** [comment]
Version bump added, good catch
